tests/test_tasks/test_clustering_task.py

```diff
@@ -43,4 +43,5 @@ def test_upload_task(self):
         )
 
         task_id = task.publish()
+        # not tracking upload for delete since _delete_entity called end of function
```

Comment:
Hm, could you change the behavior to be consistent?

Reply:
Alright, I shall then remove the `_delete_entity()` call that existed from before.